From: Francois Gouget <fgouget@codeweavers.com>
Subject: [PATCH] cmd: Match the Windows ver output.
Message-Id: <E1l4rZi-00052L-I2@amboise>
Date: Wed, 27 Jan 2021 21:38:02 +0100 (CET)

Some programs parse this string to get the Windows version!

Signed-off-by: Francois Gouget <fgouget@codeweavers.com>
---

cmd seems to use the same string for "cmd /c ver" and the first line of 
"cmd /?".

On Windows 10 neither string is translated in the locales I checked 
(French, Japanese, Chinese, Hebrew). That does not prove it's never 
translated but it does look like it isn't.

So maybe that justifies moving the string out of the resource file so it 
does not appear in the PO files. If so I can resubmit a modified 
version.


 programs/cmd/cmd.rc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/programs/cmd/cmd.rc b/programs/cmd/cmd.rc
index 11111111111..11111111111 100644
--- a/programs/cmd/cmd.rc
+++ b/programs/cmd/cmd.rc
@@ -393,7 +393,7 @@ Enter HELP <command> for further information on any of the above commands.\n"
   WCMD_NOPATH, "PATH not found\n"
   WCMD_ANYKEY,"Press any key to continue... "
   WCMD_CONSTITLE,"Wine Command Prompt"
-  WCMD_VERSION,"Microsoft Windows %1!S!\n"
+  WCMD_VERSION,"Microsoft Windows [Version %1!S!]\n"
   WCMD_MOREPROMPT, "More? "
   WCMD_LINETOOLONG, "The input line is too long.\n"
   WCMD_VOLUMELABEL, "Volume in drive %1!c! is %2\n"

-- 
2.39.0

